roxctl/generate/netpol/cta.go

```diff
@@ -1,28 +1,59 @@
+// Package netpol provides primitives for command 'roxctl generate netpol'
 package netpol
 
 import (
 	"os"
 	"path/filepath"
 	"strings"
 
-	"github.com/np-guard/cluster-topology-analyzer/pkg/controller"
+	npguard "github.com/np-guard/cluster-topology-analyzer/pkg/controller"
 	"github.com/pkg/errors"
 	"github.com/stackrox/rox/pkg/protoconv/networkpolicy"
 	v1 "k8s.io/api/networking/v1"
 )
 
+var errNPGErrorsIndicator = errors.New("there were errors during execution")
```

Comment:
Shouldn't there be an additional error for warnings, i.e. `there were warnings during execution`?
Looking at the documentation and also the logic, it's weird that we state there were errors during execution when it is warnings that we fail on with `--strict`.

Reply:
Done in [3ae9aad](https://github.com/stackrox/stackrox/pull/2637/commits/3ae9aadf73c9b5e8e039e1737b2f9beafde03475)